Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:20-slim
+
+RUN apt update && apt install --no-install-recommends -y build-essential
+
+WORKDIR /app
+
+COPY . .
+
+RUN npm ci
+
+# Using the 1000:1000 user is recommended for VSCode dev containers
+# https://code.visualstudio.com/remote/advancedcontainers/add-nonroot-user
+USER node
+
+ENV FILE=""
+ENV COUCH_URL=""
+
+VOLUME /app/test-data
```

Comment:
Do we actually need this line for anything?  (Since this volume must always be explicitly mapped by the user anyway....)

Reply:
After removing it, it's funning fine! Thanks for the tip